docs/local-development.md

```diff
@@ -0,0 +1,15 @@
+# Local Development
+
+If you are going to make code changes to this driver, and you'd like to test them against an OpenShift cluster, run the 
+`build-image` make target in this repository to capture those changes in an image reference whose remote registry and repository you can push
+to, and then employ the steps described in the [Openshift CSI Driver for Shared Resources Operator Quick Start](https://github.com/openshift/csi-driver-shared-resource-operator/blob/master/README.md#quick-start),
+where you set the `DRIVER_IMAGE` environment variable to the image reference created by your `make build-image` against
+your local clone of this repository.
+
+See that operator's [quick start guide](https://github.com/openshift/csi-driver-shared-resource-operator#quick-start) for 
+complete details.
+
+NOTE: changes to API objects that act in concert with the driver (RBAC, CSI Driver definition, service, serviceaccounts, etc)
+are defined at [https://github.com/openshift/csi-driver-shared-resource-operator/tree/master/assets](https://github.com/openshift/csi-driver-shared-resource-operator/tree/master/assets).
+If your changes need adjustments to those objects, you'll need to use `make deploy` to rollout a new version of the operator,
+per the same quick start guide.
```

Comment:
I wonder if the "RHTAP short cut" we devised to bypass tech preview and the operator is worth curating as part of upstreaming this @adambkaplan 

i.e. just apply the yaml noted in our old kustomization file (which has been deleted in the latest version):  https://github.com/gabemontero/infra-deployments/blob/d019b9be148dfa6fbf4ce4d0980389484dfd0ba5/components/shared-resources/kustomization.yaml

@sayan-biswas @avinal FYI

Reply:
For upstreaming a Helm chart makes sense. But we can wait and see what the SIG requests/requires.

Reply:
sounds good .... just something to keep in mind / in our back pocket